acceptable/tests/_fixtures.py

```diff
@@ -10,6 +17,9 @@
 
 
 def clean_up_module(name, old_syspath=None):
+    if PY2 and isinstance(name, str):
+        name = name.encode('utf8')
```

Comment:
Maybe `from future.utils import bytes_to_native_str as n` and `sys.modules.pop(n(name))`?